trade-finance-manager-ui/server/api.js

```diff
@@ -1251,6 +1258,27 @@ const updateDealCancellation = async (dealId, cancellationUpdate, userToken) =>
   }
 };
 
+/**
+ * Updates the deal cancellation object on a TFM MIN or AIN deal
+ * @param {string} dealId - The deal ID
+ * @param {string} userToken - The user token
+ * @returns {Promise<Partial<import('@ukef/dtfs2-common').TfmDealCancellation>>}
+ */
+const getDealCancellation = async (dealId, userToken) => {
+  try {
+    const response = await axios({
+      method: 'get',
+      url: `${TFM_API_URL}/v1/deals/${dealId}/cancellation`,
+      headers: generateHeaders(userToken),
+    });
+
+    return response.data;
```

Comment:
```suggestion
    return response?.data;
```

Reply:
I dont think `response` is ever falsey?

Pasting it into a ts file and adding this
```
if (!response) {
  console.log(response);
}
```
`response` has type `never` inside the block